package.json

```diff
@@ -53,9 +58,10 @@
     "ember-cli-babel": "^5.1.5",
     "ember-computed-decorators": "^0.2.2",
     "ember-cli-htmlbars": "^1.0.1",
-    "ember-cli-sass": "^5.2.0",
     "ember-frost-theme": "^1.0.1",
-    "liquid-fire": "^0.22.0"
+    "liquid-fire": "^0.22.0",
+    "ember-cli-sass": "^5.2.0",
+    "frost-svg": "~1.1.0"
```

Comment:
You have to remove frost-svg instances from your code as well. And replace with the new frost-icon API.